src/pages/app-blog/app-blog.scss

```diff
@@ -24,98 +29,193 @@
-        }
+      @include media-breakpoint-down(sm) {
+        background: #fff;
+        border: 1px solid #c75300;
+        padding: 4px 15px;
+      }
 
-        span {
-          font-size: 14px;
-          text-transform: uppercase;
-          color: $gray-dark;
-          font-weight: bold;
+      span {
+        font-size: 14px;
+        text-transform: capitalize;
+        color: $gray-dark;
+        font-weight: bold;
 
-          i {
-            color: #000;
-          }
+        i {
+          color: #000;
         }
       }
+    }
 
-      .contact-icons-label {
-        margin: 0;
-      }
+    .contact-icons-label {
+      margin: 0;
+      font-weight: 700;
+      letter-spacing: 1.02px;
+      font-size: 16px;
+      color: #9baaab;
+    }
 
-      .contact-icons {
-        width: 100%;
-      }
+    .contact-icons {
+      width: 100%;
+    }
 
-      .contact-icon {
-        margin: 10px;
-        img {
-          width: 20px;
-        }
+    .contact-icon {
+      margin-right: 12px;
+      img {
+        width: 20px;
       }
     }
   }
 }
 
 .blog-filters {
   position: sticky;
-  top: 70px;
   background: #fff !important;
   z-index: 1;
 
   .blog-filters-nav {
     background-color: unset;
 
+    @include media-breakpoint-up(lg) {
+      margin-left: 10%;
+      margin-right: 10%;
+    }
+    @include media-breakpoint-down(lg) {
+      margin: 0;
+    }
+
+    .blog-search-group {
+      @include media-breakpoint-up(xl) {
+        padding-left: 66.6px; // The number of the beast
+      }
+      @include media-breakpoint-down(md) {
+        width: calc(100% - 16px);
+        margin: 8px;
+      }
+      width: 100%;
+    }
+
+    .blog-search-input {
+      padding: 4px 4px 4px calc(2.4rem + 2px);
+      border: none;
+      background-color: #f4f5f9;
+      box-shadow: inset -1px -1px 1px 0 rgba(0, 0, 0, 0.15);
+      border-radius: 3px;
+      width: 100%;
+      height: 32px;
+      font-family: Muli;
+      color: #5d534e;
+      font-size: 13px;
+      font-weight: 300;
+      line-height: 18px;
+
+      @include media-breakpoint-down(sm) {
+        box-shadow: none;
+        border: 1px solid #ebecef;
+        border-radius: 6px;
+      }
+    }
+
+    .blog-search-input-top {
+      padding: 4px 4px 4px calc(2.4rem + 2px);
+      border: none;
+      background-color: #f4f5f9;
+      border-radius: 3px;
+      width: 85%;
+      height: 32px;
+      font-family: Muli;
+      color: #5d534e;
+      font-size: 13px;
+      font-weight: 300;
+      line-height: 18px;
+      border: 1.5px solid rgba(0, 0, 0, 0.15);
+    }
+
+    .blog-search-icon {
+      position: absolute;
+      margin-top: 10px;
+      margin-left: 10px;
+      color: #615853;
+      @include media-breakpoint-down(md) {
+        position: initial;
+        margin-top: 0;
+        margin-left: 0;
+      }
+    }
+
+    .blog-search-icon-top {
+      position: absolute;
+      margin-top: 10px;
+      margin-left: 10px;
+      color: #615853;
```

Comment:
Do we have a global variable for this color? If not, we should probably add it for ease of use. I've seen it a few times within this file.